django/template/autoreload.py

```diff
@@ -16,13 +17,13 @@ def get_template_directories():
         if not isinstance(backend, DjangoTemplates):
             continue
 
-        items.update(to_path(dir).absolute() for dir in backend.engine.dirs)
+        items.update(Path.cwd() / to_path(dir) for dir in backend.engine.dirs)
```

Comment:
Why not `resolve()`? :thinking:

Reply:
It doesn't work on Windows. :| Non-existent files simple **don't** get resolved. 

```
>>> from pathlib import Path
>>> file = Path('new.txt')
>>> file.exists()
False
>>> file.resolve()
WindowsPath('new.txt')
>>> file.absolute()
WindowsPath('d:/home/new.txt')
>>> file.touch()
>>> file.resolve()
WindowsPath('D:/home/new.txt')
>>> file.unlink()
>>> file.resolve()
WindowsPath('new.txt')
```

This is presumably why @hramezani went for `absolute()` (right?) but that's undocumented and shouldn't be used. 

> Path.absolute() is undocumented and thus unsupported, so use at your own peril. — Brett Cannon. 

* https://discuss.python.org/t/pathlib-absolute-vs-resolve/2573
* https://bugs.python.org/issue38671
* https://bugs.python.org/issue39090
* **Slightly Misleading in that it suggests `absolute()`**: https://stackoverflow.com/questions/42513056/how-to-get-absolute-path-of-a-pathlib-path-object

Reply:
But should we worry about non-existent directories? we can use `template_tests/templates/first` and `template_tests/templates/second` in tests.

Reply:
They'll still be dependent on `cwd` I think... 🤔